app/main/views/questionnaire.py

```diff
@@ -47,15 +53,17 @@ def add_cache_control(response):
 @questionnaire_blueprint.route('<location>', methods=["GET"])
 @login_required
 def get_questionnaire(eq_id, form_type, collection_id, location):
-    return render_page(location, True)
+    g.questionnaire_manager.build_state(location, get_answers(current_user))
```

Comment:
I kind of feel we shouldn't be accessing flasks globals in this way. Flask has pretty clear guidance on using the app context http://flask.pocoo.org/docs/0.11/appcontext/. 

I'm aware this has crept in elsewhere, but I think we should be creating accessors as per the questionnaire store approach in globals.py.